Formula/php.rb

```diff
@@ -0,0 +1,316 @@
+      end
+
+      if !File.exist?(config_path+"php-fpm.d/www.conf") && File.exist?(config_path+"php-fpm.d/www.conf.default")
+        mv(config_path+"php-fpm.d/www.conf.default", config_path+"php-fpm.d/www.conf")
+      end
+
+      unless File.exist?(config_path+"php-fpm.conf")
+        if File.exist?("sapi/fpm/php-fpm.conf")
+          config_path.install "sapi/fpm/php-fpm.conf"
+        end
+
+        if File.exist?("sapi/cgi/fpm/php-fpm.conf")
+          config_path.install "sapi/cgi/fpm/php-fpm.conf"
+        end
+
+        inreplace config_path+"php-fpm.conf" do |s|
+          s.sub!(/^;?daemonize\s*=.+$/, "daemonize = no")
+        end
+      end
+
+      rm_f("#{config_pear}-backup") if File.exist? "#{config_pear}-backup"
+      rm_f("#{user_pear}-backup") if File.exist? "#{user_pear}-backup"
+      rm_f("#{config_pearrc}-backup") if File.exist? "#{config_pearrc}-backup"
+      rm_f("#{user_pearrc}-backup") if File.exist? "#{user_pearrc}-backup"
+    rescue StandardError
+      mv("#{config_pear}-backup", config_pear) if File.exist? "#{config_pear}-backup"
+      mv("#{user_pear}-backup", user_pear) if File.exist? "#{user_pear}-backup"
+      mv("#{config_pearrc}-backup", config_pearrc) if File.exist? "#{config_pearrc}-backup"
+      mv("#{user_pearrc}-backup", user_pearrc) if File.exist? "#{user_pearrc}-backup"
+      raise
+    end
+  end
+
+  def caveats
+    s = []
+
+    s << <<-EOS.undent
+      To enable PHP in Apache add the following to httpd.conf and restart Apache:
+          LoadModule php7_module #{HOMEBREW_PREFIX}/opt/php#{version.to_s[0..2].delete(".")}/libexec/apache2/libphp7.so
+
+          <FilesMatch \.php$>
+              SetHandler application/x-httpd-php
+          </FilesMatch>
+
+      Finally, check DirectoryIndex includes index.php
+          DirectoryIndex index.php index.html
+    EOS
+
+    s << <<-EOS.undent
+      The php.ini file can be found in:
+          #{etc}/php/#{version.to_s.split(".")[0..1].join(".")}/php.ini
+    EOS
+
+    s.join "\n"
+  end
+
+  def post_install
+    # Belongs to fpm config
+    (prefix/"var/log").mkpath
+    touch prefix/"var/log/php-fpm.log"
+
+    # Fix pecl/pear permissions
+    chmod 0755, lib/"php/.channels"
+    chmod 0644, Dir.glob(lib/"php/.channels/**/*")
+
+    %w[
+      php/.depdblock
+      php/.filemap
+      php/.depdb
+    ].each do |f|
+      chmod 0644, lib/f
+    end
+
+    php_ini = "#{etc}/php/#{version.to_s.split(".")[0..1].join(".")}"
+    system bin/"pear", "config-set", "php_ini", php_ini, "system"
+
+    %w[
+      ldap
+      mcrypt
+      tidy
+      opcache
+    ].each do |e|
+      config_path = (etc/"php/#{version.to_s.split(".")[0..1].join(".")}/conf.d/ext-#{e}.ini")
+      extension_type = (e == "opcache") ? "zend_extension" : "extension"
+      if File.exist? config_path
+        inreplace config_path, /^#{extension_type}=.*$/, "#{extension_type}=#{Utils.popen_read("php-config --extension-dir").chomp}/#{e}.so"
```

Comment:
This needs a more reliable way to update the link to the extension since people might put a # in front of the line to disable a specific module. This inreplace line will not detect that and fail.